gamemodes/terrortown/gamemode/client/cl_targetid.lua

```diff
@@ -491,23 +492,25 @@ function HUDDrawTargetIDDoors(tData)
 
 	if ent:UseOpensDoor() and not ent:TouchOpensDoor() then
 		tData:SetSubtitle(ent:IsDoorOpen() and ParT("door_close", key_params) or ParT("door_open", key_params))
+		tData:SetKey(input.GetKeyCode(key_params.usekey))
 	elseif not ent:UseOpensDoor() and ent:TouchOpensDoor() then
 		tData:SetSubtitle(TryT("door_open_touch"))
+		tData:AddIcon(
```

Comment:
Why is this formatted? As no other function has such a format (only tables etc)

Reply:
This is the same style as everywhere else in this file